packages/cli/lib/cli.js

```diff
@@ -29,16 +30,18 @@ class Cli {
     const command = args._[0] || 'help';
 
     switch (command) {
+      case 'curls':
+        return require('./cmds/curls')(args, this.logger_);
       case 'help':
         return require('./cmds/help')(args, this.logger_);
-      case 'version':
-        return require('./cmds/version')(args, this.logger_);
+      case 'optimize':
+        return new OptimizeCmd().run(args, this.logger_);
```

Comment:
I'm wondering if we should make all the commands to use the same format as this. I do think it reads better than requiring inline. The downside seems to be requiring upfront makes the code to run the command load sooner, which may lead to slower "boot time" for the CLI?

Reply:
Good point. If inlined requiring the command.